# TAP results for testing/code/test_excinfo.py
ok 1 - testing/code/test_excinfo.py::test_excinfo_simple
ok 2 - testing/code/test_excinfo.py::test_excinfo_getstatement
ok 3 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_entries
ok 4 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_entry_getsource
ok 5 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_entry_getsource_in_construct
ok 6 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_cut
ok 7 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_cut_excludepath
ok 8 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_filter
ok 9 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_filter_selective[<lambda>-True]
ok 10 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_filter_selective[<lambda>-False]
ok 11 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_filter_selective[tracebackhide2-True]
ok 12 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_filter_selective[tracebackhide3-False]
ok 13 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_recursion_index
ok 14 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_only_specific_recursion_errors
ok 15 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_no_recursion_index
ok 16 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_messy_recursion # SKIP could not import 'decorator'
ok 17 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_getcrashentry
ok 18 - testing/code/test_excinfo.py::TestTraceback_f_g_h.test_traceback_getcrashentry_empty
ok 19 - testing/code/test_excinfo.py::test_excinfo_exconly
ok 20 - testing/code/test_excinfo.py::test_excinfo_repr
ok 21 - testing/code/test_excinfo.py::test_excinfo_str
ok 22 - testing/code/test_excinfo.py::test_excinfo_errisinstance
ok 23 - testing/code/test_excinfo.py::test_excinfo_no_sourcecode
ok 24 - testing/code/test_excinfo.py::test_excinfo_no_python_sourcecode # SKIP could not import 'jinja2'
ok 25 - testing/code/test_excinfo.py::test_entrysource_Queue_example
ok 26 - testing/code/test_excinfo.py::test_codepath_Queue_example
ok 27 - testing/code/test_excinfo.py::test_match_succeeds
ok 28 - testing/code/test_excinfo.py::test_match_raises_error
ok 29 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_source
ok 30 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_source_excinfo
ok 31 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_source_not_existing
ok 32 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_many_line_source_not_existing
ok 33 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_source_failing_fullsource
ok 34 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_local
ok 35 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_tracebackentry_lines
ok 36 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_tracebackentry_lines2
ok 37 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_tracebackentry_lines_var_kw_args
ok 38 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_tracebackentry_short
ok 39 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_tracebackentry_no
ok 40 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_traceback_tbfilter
ok 41 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_traceback_short_no_source
ok 42 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_traceback_and_excinfo
ok 43 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_traceback_with_invalid_cwd
ok 44 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_excinfo_addouterr
ok 45 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_excinfo_reprcrash
ok 46 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_repr_traceback_recursion
ok 47 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_reprexcinfo_getrepr
ok 48 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_reprexcinfo_unicode
ok 49 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_toterminal_long
ok 50 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_toterminal_long_missing_source
ok 51 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_toterminal_long_incomplete_source
ok 52 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_toterminal_long_filenames
ok 53 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions0]
ok 54 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions1]
ok 55 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions2]
ok 56 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions3]
ok 57 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions4]
ok 58 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions5]
ok 59 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions6]
ok 60 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions7]
ok 61 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions8]
ok 62 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions9]
ok 63 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions10]
ok 64 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions11]
ok 65 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions12]
ok 66 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions13]
ok 67 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions14]
ok 68 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions15]
ok 69 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions16]
ok 70 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions17]
ok 71 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions18]
ok 72 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions19]
ok 73 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions20]
ok 74 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions21]
ok 75 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions22]
ok 76 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_format_excinfo[reproptions23]
ok 77 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_traceback_repr_style
ok 78 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_exc_chain_repr # SKIP condition: sys.version_info[0] < 3
ok 79 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_exc_repr_with_raise_from_none_chain_suppression # SKIP condition: sys.version_info[0] < 3
ok 80 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_exc_chain_repr_without_traceback[cause-The above exception was the direct cause of the following exception:] # SKIP condition: sys.version_info[0] < 3
ok 81 - testing/code/test_excinfo.py::TestFormattedExcinfo.test_exc_chain_repr_without_traceback[context-During handling of the above exception, another exception occurred:] # SKIP condition: sys.version_info[0] < 3
ok 82 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[None-short]
ok 83 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[None-long]
ok 84 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[utf8-short]
ok 85 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[utf8-long]
ok 86 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[utf16-short]
ok 87 - testing/code/test_excinfo.py::test_repr_traceback_with_unicode[utf16-long]
ok 88 - testing/code/test_excinfo.py::test_cwd_deleted
ok 89 - testing/code/test_excinfo.py::test_exception_repr_extraction_error_on_recursion
ok 90 - testing/code/test_excinfo.py::test_no_recursion_index_on_recursion_error
1..90
